tests/cypress/integration/actions_tasks_objects/case_35_search_task_feature.js

```diff
@@ -16,12 +16,33 @@ context('Search task feature.', () => {
         cy.contains('.cvat-item-task-name', taskName).should(result);
     }
 
+    before(() => {
+        cy.openTask(taskName);
+        cy.assignTaskToUser(Cypress.env('user')); // Assigne a task to an ures to check filter
+        cy.goToTaskList();
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.openTask(taskName);
+        cy.assignTaskToUser('');
+    });
+
     describe(`Testing case "${caseId}"`, () => {
-        it('Type to task search some field and check result.', () => {
+        it('Tooltip task filter contain all the possible options.', () => {
+            cy.get('.cvat-task-page-search-task').trigger('mouseover');
+            cy.get('.cvat-tasks-search-tooltip').should('be.visible').invoke('text').then(($tooltipText) => {
+                for (const i of ['owner: admin', 'assignee: employee', 'name: training', 'mode: annotation', 'status: annotation', 'id: 5']) {
+                    expect($tooltipText).contain(i);
+                }
+            });
+        });
```

Comment:
I do not think we should test tooltips in this way. Its content can be changed and probably we do not want the tests to be failed in this case. IMHO checking if a tooltip appears would be enough.

Reply:
Removed the text check.